### PR TITLE
Fix broken import of BaseSettings for pydantic v2

### LINKED ISSUE #81: Fastapi version needs to be <0.100
pip installing current version wont start because of errors with Pydantic 2 (BaseSettings has been moved), fastapi version should be set to less the 0.100 until grove works for pydantic 2.

### PR DESCRIPTION
**Pull request by loichuder**
Fix #81 

It was indeed an import to change.

https://fastapi.tiangolo.com/advanced/settings/#pydantic-settings

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -27,7 +27,8 @@ install_requires =
 [options.extras_require]
 fastapi = 
     fastapi
-	uvicorn
+	pydantic-settings
```
Comment (t20100) [inline setup.cfg:30]: So this forces to use `pydantic` v2:

https://github.com/pydantic/pydantic-settings/blob/4f794c7442a3c71d6d3280c888cdd8c288350543/pyproject.toml#L42-L45

**Review by t20100 — APPROVED** [review]
LGTM.

Maybe worth bumping the major number for next release since now it requires `pydantic` v2.

**Comment by loichuder** [comment]
> LGTM.
> 
> Maybe worth bumping the major number for next release since now it requires `pydantic` v2.

If this is the case, we can move to Python 3.8 for me to avoid chasing this annoying CI bug.

File: setup.cfg (hunk 1 above)
Comment (loichuder) [reply setup.cfg:30]: Do you think we should state the dependency explicitly here ?

Comment (t20100) [reply setup.cfg:30]: Yes maybe, solving dependency issues quickly becomes a nightmare.

**Comment by loichuder** [comment]
> > we can move to Python 3.8
> 
> Sure! Python3.7 has reached end-of-life (+ some packages like `numpy` starts dropping Python3.8...)

Let's do this in another PR. Thanks for the review :+1: !

**Pull request merged**